rio_tiler_pds/landsat/utils.py

```diff
@@ -129,32 +183,49 @@ def sceneid_parser(sceneid: str) -> Dict:
     elif meta["sensor"] == "M":
         sensor_name = "mss"
 
+    # S3 paths always use oli-tirs
+    _sensor_name = sensor_name
+    if _sensor_name in ["oli", "tirs"]:
+        _sensor_name = "oli-tirs"
+
     meta["sensor_name"] = sensor_name
+    meta["_sensor_name"] = _sensor_name
```

Comment:
can we rename `_sensor_name` to `sensor_dir_name` ? it's a bit confusing

Reply:
How about `_sensor_s3_prefix`? :smile: